website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -155,4 +155,47 @@ Row placement determines the visual styling based on where the row is placed rel
 - We recommend using `medium` cell density by default.
 - If content is complex or a smaller data set (e.g., a Table of basic user data), `tall` cell density allows for more breathing room surrounding the content.
 - If content is largely string/text-based, `short` allows for more content to be displayed within the page.
-- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
\ No newline at end of file
+- While denser content allows for more rows to be displayed within a single page, it also makes comprehension and scanning more difficult.
+
+## Multi-Selection
+
+Multi-selection enables users to select multiple rows in a table, enhancing efficiency and minimizing errors. This feature facilitates bulk operations, boosting productivity and optimizing data management.
```

Comment:
```suggestion
Multi-selection enables users to select multiple rows in a table, facilitating bulk operations, actions, and editing.
```

I still think this point around productivity and data management is a bit irrelevant (at least for this type of documentation). Though curious what other folks thing.